assets/src/containers/Course.js

```diff
@@ -16,6 +16,16 @@ function Course (props) {
   const [loaded, error, courseInfo] = useCourseInfo(courseId)
   const [sideDrawerState, setSideDrawerState] = useState(false)
 
+  const toggleDrawer = (open) => event => {
```

Comment:
This can be separated to its own component and imported and used here

Reply:
This will be resolved later